### PR TITLE
LRAC-7662 Prevent send invalid events

### PR DESCRIPTION
**Pull request by fcodias**
https://issues.liferay.com/browse/LRAC-7662

### CODE REVIEW

**Comment by fcodias** [comment]
cc @kienD @JhonattasFerreira

**Comment by fcodias** [comment]
ci:test:sf

File: modules/apps/analytics/analytics-client-js/src/utils/constants.js [hunk 1]
```diff
@@ -79,3 +79,9 @@ export const READ_WORDS_PER_MIN = 265;
 export const TRACK_DEFAULT_OPTIONS = {
 	applicationId: 'CustomEvent',
 };
+
+//Validation
```
Comment (andreldm) [inline modules/apps/analytics/analytics-client-js/src/utils/constants.js]: sf

File: modules/apps/analytics/analytics-client-js/src/utils/constants.js [hunk 2]
```diff
@@ -79,3 +79,9 @@ export const READ_WORDS_PER_MIN = 265;
...
+
+export const VALIDATION_MAXIMUM_LENGTH_DEFAULT = 255;
```
Comment (andreldm) [inline modules/apps/analytics/analytics-client-js/src/utils/constants.js]: `VALIDATION_PROPERTY_NAME_MAXIMUM_LENGTH` ?

File: modules/apps/analytics/analytics-client-js/src/utils/constants.js [hunk 3]
```diff
@@ -79,3 +79,9 @@ export const READ_WORDS_PER_MIN = 265;
...
+
+export const VALIDATION_MAXIMUM_PROPERTY_VALUE_LENGTH = 1024;
```
Comment (andreldm) [inline modules/apps/analytics/analytics-client-js/src/utils/constants.js]: `VALIDATION_PROPERTY_VALUE_MAXIMUM_LENGTH` ?

**Comment by fcodias** [comment]
Updated @andreldm, thanks!!

**Comment by fcodias** [comment]
ci:test:sf

**Comment by andreldm** [comment]
Untested, but code wise seems good, thanks!

**Comment by andreldm** [comment]
ci:forward

**Comment by alexcandidos** [comment]
ci:forward

**Comment by alexcandidos** [comment]
Asking CI again since the last command does not seem to have triggered CI as expected.

**Pull request closed without merging**